Microsoft.Toolkit.Uwp.UI.Controls.Layout/ListDetailsView/ListDetailsView.Properties.cs

```diff
@@ -296,34 +381,34 @@ public DataTemplate NoSelectionContentTemplate
         }
 
         /// <summary>
-        /// Gets the current visual state of the control
+        /// Gets or sets gets the current visual state of the control.
         /// </summary>
         public ListDetailsViewState ViewState
         {
             get { return (ListDetailsViewState)GetValue(ViewStateProperty); }
-            private set { SetValue(ViewStateProperty, value); }
+            set { SetValue(ViewStateProperty, value); }
```

Comment:
This isn't really something that can be set. It's determined based on the available space and if something is selected or not

Reply:
Done.